Source/Diagnostics/BTDiagnostics.H

```diff
@@ -310,6 +314,12 @@ private:
         return ( m_buffer_counter[i_buffer] == 0) ;
     }
 
+    amrex::Vector<int> m_first_flush_after_restart;
```

Comment:
Can you please add 4 doxygen strings to the 3 new member vars and 1 new function?

Reply:
I have added it in the diagnostics.H where they are virtual methods. I will add for the 3 new member vars now. 
For the functions that are virtual methods in Diagnostics.H and we override them here, should we repeat doxygen?

Reply:
I think that makes sense, no need to repeat if overridden :)